main.py

```diff
@@ -23,20 +19,17 @@ def main():
     init = tf.global_variables_initializer()
     sess.run(init)
 
-    threads = tf.train.start_queue_runners(sess=sess, coord=coord)
-    reader.start_threads(sess)
-
+    threads = reader.start_threads(sess)
     net = define_net(input_batch)
-
-    for step in range(int(1e9)):
-        # The queue is filled faster than what the main thread can unstack.
-        # That's the reason why size of the queue is almost always equal to 32.
-        print('size queue =', sess.run(reader.size()))
+    queue_size = reader.queue_size
+    for step in xrange(10000):
```

Comment:
range is ok for a demo.. but range(1e9) is too big... I didn't notice this at the beginning and this stuck my laptop two times.